src/GitHub.Exports/Extensions/VSExtensions.cs

```diff
@@ -75,5 +81,64 @@ static object GetServiceAndCache<CacheType>(IServiceProvider provider, Type type
                 cache = (CacheType)(object)ret;
             return ret;
         }
+
+        public static void AddTopLevelMenuItem(this IServiceProvider provider,
+            Guid guid,
+            int cmdId,
+            EventHandler eventHandler)
+        {
+            var mcs = provider.GetService(typeof(IMenuCommandService)) as IMenuCommandService;
+            Debug.Assert(mcs != null, "No IMenuCommandService? Something is wonky");
+            if (mcs == null)
+                return;
+            var id = new CommandID(guid, cmdId);
+            var item = new MenuCommand(eventHandler, id);
+            mcs.AddCommand(item);
+        }
+
+        public static void AddDynamicMenuItem(this IServiceProvider provider,
+            Guid guid,
+            int cmdId,
+            Func<bool> canEnable,
+            Action execute)
+        {
+            var mcs = provider.GetService(typeof(IMenuCommandService)) as IMenuCommandService;
+            Debug.Assert(mcs != null, "No IMenuCommandService? Something is wonky");
+            if (mcs == null)
+                return;
+            var id = new CommandID(guid, cmdId);
+            var item = new OleMenuCommand(
+                (s, e) => execute(),
+                (s, e) => { },
+                (s, e) =>
+                {
+                    ((OleMenuCommand)s).Visible = canEnable();
+                },
+                id);
+            mcs.AddCommand(item);
+        }
+    }
+
+    public static class ISolutionExtensions
+    {
+        public static UriString GetRepoUrlFromSolution(IVsSolution solution)
```

Comment:
It's an oversight. This method isn't being currently used, and the way it stands right now, it doesn't have valuable logic that's worth keeping it around (GetRepoFromSolution does the same exact thing), so might just kill it.